### PR TITLE
CBL-4399: Add CBL-4330 merge to Blame Ignore.

### PR DESCRIPTION
**Pull request by callumbirks**

### CODE REVIEW

**Comment by callumbirks** [comment]
This is blocked until CBL-4400 is merged. After which, I can add the commit hash for that merge to the Blame Ignore file also.

**Comment by callumbirks** [comment]
Again, if you wish to clean up your local Core Git blame using this file, you can do `git config blame.ignoreRevsFile .git-blame-ignore-revs`. I've added this instruction as a comment in the file now as well.

**Review by jianminzhao — APPROVED** [review]

**Review by pasin — APPROVED** [review]

**Pull request merged**
